app/controllers/admin/users_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Admin::UsersController < ApplicationController
+  before_action :authenticate_user!
   def index
     @user = User.all
```

Comment:
This endpoint should be available only for admins. Please create a method for checking this role and use this method as before_action in this file